hello-world/README.md

```diff
@@ -11,27 +11,27 @@ Later on, you'll deploy a Python app to act as the publisher. The architecture d
 ## Prerequisites
 This quickstart requires you to have the following installed on your machine:
 - [Docker](https://docs.docker.com/)
-- [Node.js version 8 or greater](https://nodejs.org/en/) 
-- [Python 3.x](https://www.python.org/downloads/): Note: When running this quickstart on Windows, it best to install Python from python.org rather than from the Windows store. 
+- [Node.js version 8 or greater](https://nodejs.org/en/)
+- [Python 3.x](https://www.python.org/downloads/): Note: When running this quickstart on Windows, it best to install Python from python.org rather than from the Windows store.
 - [Postman](https://www.getpostman.com/) [Optional]
 
-## Step 1 - Setup Dapr 
+## Step 1 - Setup Dapr
 
 Follow [instructions](https://docs.dapr.io/getting-started/install-dapr/) to download and install the Dapr CLI and initialize Dapr.
 
 ## Step 2 - Understand the code
 
-Now that Dapr is set up locally, clone the repo, then navigate to the Hello World quickstart: 
+Now that Dapr is set up locally, clone the repo, then navigate to the node version of the Hello World quickstart:
```

Comment:
```suggestion
Now that Dapr is set up locally, clone the repo, then navigate to the Node.js version of the Hello World quickstart:
```

Reply:
Oh for @greenie-msft, should this get pushed to the feature_newquickstarts branch, /tutorials/.. folder too?